library/ts-rollup/README.md

```diff
@@ -1,16 +1,41 @@
-# ♻️ Usage
+# Environment with TypeScript and Rollup
+
+## ♻️ Usage
+
+The production bundle is available as [ESM bundle](./dist/bingo-generator.esm.js).
+
+### Initialize the library
+```javascript
+    import { Bingo } from 'bingo-generator';
+    
+    // Initialize bingo-generator
+    // 'bingo-container' is the id of the HTMLElement that renders the Bingo
+    const bingo = new Bingo('bingo-container');
+```
+
+### Generate a new grid with random numbers on each square
+```javascript
+    bingo.resetGrid();
+```
+The result should be something like this:
+  <p align="center">
+    <img title="Grid" src="../../images/grid.png">
+  </p>
+
+## Technical part
```

Comment:
⚠️ the code is simple enough to not need this info. I don't want to backport this changes to the webpack part

Reply:
In my opinion, for someone who knows the code, yes, it's simple to understand.
But, when I arrive on a new library code, I like to have a minimum to know where wtart & where are the APIs.

Reply:
I can do the change on Webpack part, if it's the problem.

Reply:
yes, let's leave the explanation, not everyone may be familiar with jest-image-snapshot